Upcoming-iCal-Events.widget/main.coffee

```diff
@@ -1,13 +1,41 @@
 # Author: Connor Beardsmore <connor.beardsmore@gmail.com>
 # Last Modified: 22/03/18
 
+# Customisations
+# Refresh Frequency
+REFRESH_FREQUENCY = "1h"
+# Number of days shown from today.
+NO_DAYS_TO_SHOW = 2
+# Show which calendar you pulled from before event name.
+SHOW_CALENDER = false
+# Ignore specific calendars.
+IGNORE_CALENDERS = [
+    'name/UUID of thecalendar to ignore',
+    'other calendar etc'
+]
+# Show full date including time.
+SHOW_DATE_TIME = true
+# Characters after this value will be replaced with ...
+MAX_CHARACTERS = 50
+# Use date for day after tomorrow.
+USE_LATER_DATE = false
```

Comment:
nit; could be confusing without context of the PR, maybe something like `USE_DATE_AS_SUBHEADING`

Reply:
Renamed.